lib/reporters/mini.js

```diff
@@ -18,22 +23,45 @@ function MiniReporter() {
 module.exports = MiniReporter;
 
 MiniReporter.prototype.start = function () {
-	return '';
+	var self = this;
+	this.interval = setInterval(function () {
+		self.spinnerFrame = (self.spinnerFrame + 1) % self.spinnerFrames.length;
+		self.write(self.spinnerChar() + self.currentTest);
+	}, this.spinnerInterval);
+	return this.spinnerChar();
 };
 
 MiniReporter.prototype.reset = function () {
+	this.clearInterval();
 	this.passCount = 0;
 	this.failCount = 0;
 	this.skipCount = 0;
 	this.rejectionCount = 0;
 	this.exceptionCount = 0;
 	this.currentStatus = '';
+	this.currentTest = '';
 	this.statusLineCount = 0;
+	this.spinnerFrame = 0;
 	this.lastLineTracker = lastLineTracker();
 };
 
+MiniReporter.prototype.spinnerChar = function () {
+	return this.spinnerFrames[this.spinnerFrame];
+};
+
+MiniReporter.prototype.clearInterval = function () {
+	if (this.interval) {
```

Comment:
Is there really any point in guarding this? `clearInterval` gracefully accepts anything.